src/tiled/wangfiller.h

```diff
@@ -52,15 +55,19 @@ class WangFiller
     Cell findFittingCell(const TileLayer &back,
                          const TileLayer &front,
                          const QRegion &fillRegion,
-                         QPoint point) const;
+                         QPoint point,
+                         StaggeredRenderer *staggeredRenderer = nullptr,
+                         Map::StaggerAxis staggerAxis = Map::StaggerX) const;
```

Comment:
Instead of adding these parameters to each `WangFiller` method, passing them around on each call, consider adding them to the `WangFiller` constructor. And to avoid having to keep a `WangFiller` instance up-to-date, just allocate it on the stack when you're going to call its methods. This is similar to how `TilePainter` is used.